### ISSUE TITLE
Contour lines not showing

### ISSUE THREAD

**Issue by Nine-Tailed** [osmandapp/OsmAnd#16931]
### Description
Contour lines are no longer displayed.

### Your Environment
OsmAnd Version: OsmAnd~ 4.5.0#35732m, Дата выпуска: 2023-04-07
Android version: 13
Device model: poco f3

**Comment by omawnakw** [comment]
Indeed. Starting from that nightly build controur lines and tourist routes don't show up in OpneGL mode. 
Still an issue with the latest OsmAnd~ 4.5.0#35733m, Дата выпуска: 2023-04-10
Samsung Tab S8 SM-X700, Samsung Galaxy Note 9
